scripts/canflash/README.md

```diff
@@ -1 +1,20 @@
 # CAN Flash
+
+## How to run the cantransfer.sh script
+
+### Check if there is a connection with the pi
+
+Type ``ping [ip address]`` in terminal, where ip address is the ip address of the pi.
+
+### Running the script
+
+Run the script by using the command ``sh cantransfer.sh 'bin_files' 'pi_username' 'host' 'destination_path``.
```

Comment:
Would recommend `host` is changed to `ip address`. Keeps it consistent with the above `ping` and I don't see us connecting to the pi's through other hostnames at the moment.